telepresence-configure.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+FLASK_EXECUTABLE=`pipenv --venv`/bin/flask
+
+echo -e "\nTo start/restart the flask development server, run the following command (which was also copied into your pasteboard):\n"
+echo -e "SSL_CERT_FILE=$TELEPRESENCE_ROOT$SSL_CERT_FILE \\
+REQUESTS_CA_BUNDLE=$TELEPRESENCE_ROOT$REQUESTS_CA_BUNDLE \\
+FLASK_DEBUG=1 \\
+FLASK_APP=app:app \\
+HOST_NAME=$HOST_NAME \\
+OAUTHLIB_INSECURE_TRANSPORT=1 \\
+${FLASK_EXECUTABLE} run" | tee >(pbcopy)
```

Comment:
A quick google of `pbcopy` indicates it is a mac-only thing. So I think we should remove it from here. Manually copying and pasting the command is still ok.

Reply:
or we can leave this in as long as it does not crash the whole script completely on linux and also indicate in the instructions that you can skip the manual copy step only on mac